bin/lib/zopen-build

```diff
@@ -207,7 +193,7 @@ printSyntax()
   echo "  -vv: run in very verbose mode (sets environment variables V=1 and VERBOSE=1)"
   echo "  -u|--upgradedeps: upgrade all dependencies by running zopen install"
   echo "  --buildtype: release|debug. The default is release"
-  echo "  --comp: xl|clang.  The compiler used for building.  The default is xl."
+  echo "  --comp: <compiler dependency>.  The compiler(s) used for building.  The default is comp_xlclang."
```

Comment:
```suggestion
  echo "  --comp <compiler dependency>:  The compiler(s) used for building.  The default is xlclang."
```
Move the ':' to be clearer about option syntax.  Need to do this for --buildtype too.
Also have zopen-build add the "comp_".